py/server/deephaven/experimental/s3.py

```diff
@@ -84,6 +89,28 @@ def __init__(self,
             num_concurrent_write_parts (int): the maximum number of parts that can be uploaded concurrently when writing
                 to S3 without blocking, defaults to 64. Setting a higher value may increase throughput, but may also
                 increase memory usage.
+            profile_name (str): the default profile name used for configuring the default region, credentials, etc.,
+                when reading or writing to S3. If not provided, the AWS SDK picks the profile name from the
+                'aws.profile' system property, the "AWS_PROFILE" environment variable, or defaults to "default".
+                Setting a profile name assumes that the credentials are provided via this profile; if that is not the
+                case, you must explicitly set credentials.
+            config_file_path (str): the path to the configuration file to use for configuring the default region,
```

Comment:
What do these files look like?  How do I get to a spec for creating one?  etc.

Reply:
This page has examples: https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html

Reply:
The docstring should specify where a user should look for details.

Reply:
Agreed, I have already added this link in the pydoc.